tasks/agent.py

```diff
@@ -267,7 +268,14 @@ def run(
     if not skip_build:
         build(ctx, rebuild, race, build_include, build_exclude, flavor)
 
-    ctx.run(os.path.join(BIN_PATH, bin_name("agent")))
+    cmd = os.path.join(BIN_PATH, bin_name("agent"))
+    cmd += ' run -c '
+    if config_path is None:
+        dist_folder = os.path.join(BIN_PATH, "dist")
+        cmd += os.path.join(dist_folder, "datadog.yaml")
+    else:
+        cmd += config_path
+    ctx.run(cmd)
```

Comment:
We use fstring to make it easier to read, WDYT ?

```suggestion
    ctx.run(f"{agent_bin} run -c {config_path})
```